packages/create-sitecore-jss/src/templates/nextjs-personalize/src/lib/pos-resolver.ts

```diff
@@ -0,0 +1,20 @@
+import { EnvHelper } from '@sitecore-jss/sitecore-jss-nextjs';
+
+// This class is used to get point of sale identifier(s). Resolving from env variable, but it can be expanded or change in future if needed.
+export class PosResolver {
+    static resolve = (locale: string): string => {
```

Comment:
Looks like the indentation is off in this file

Reply:
@art-alexeyenko This still looks off - it appears to be using 4 spaces instead of 2 spaces